doc/api/fs.md

```diff
@@ -3982,7 +3982,7 @@ changes:
   * `string` {string}
 
 Write `string` to the file specified by `fd`. If `string` is not a string, then
-the value will be coerced to one.
+a exception will be thrown.
```

Comment:
This will be the new behavior, when #31030 get landed.

Reply:
Shouldn't this be incorporated into the existing PR instead?

Reply:
cc @BridgeAR

Reply:
Thanks for the ping, I'll cherry-pick this in #31030.

@juanarbol please feel free to leave comments in PRs in case you see that something is missing :)